src/Fuel-Tests-Core/FLBasicSerializationTest.class.st

```diff
@@ -796,7 +796,7 @@ FLBasicSerializationTest >> testTrue [
 FLBasicSerializationTest >> testUUID [
 
 	self assertSerializationEqualityOf: (UUID fromString: 'a3b64357-377a-5b41-b575-1c653084a121').
-	self assertSerializationEqualityOf: UUID nilUUID.
+	self assertSerializationEqualityOf: NilUUID new.
```

Comment:
What is the rationale behind NilUUID ?
Why a different class?

If having a nil uuid is really important, why couldn't we have `UUID nilUUID`? That way the implementation is kinda hidden (one class, two classes, the user should not care, it's easier to refactor)